dpdata/plugins/fhi_aims.py

```diff
@@ -5,6 +5,8 @@
 @Format.register("fhi_aims/output")
 class FhiMDFormat(Format):
     def from_labeled_system(self, file_name, md=True, begin = 0, step = 1, **kwargs):
+        if "converge" not in kwargs.keys():
+            kwargs["converge"] = True
```

Comment:
This should use [`dict.setdefault`](https://docs.python.org/3/library/stdtypes.html#dict.setdefault) or be directly written into the method definition line.

Reply:
updated.